src/runtime/interface.go

```diff
@@ -5,25 +5,67 @@ package runtime
 // Interfaces are represented as a pair of {typecode, value}, where value can be
 // anything (including non-pointers).
 
-import "unsafe"
+import (
+	"reflect"
+	"unsafe"
+)
 
 type _interface struct {
 	typecode uintptr
 	value    unsafe.Pointer
 }
 
 // Return true iff both interfaces are equal.
-func interfaceEqual(x, y _interface) bool {
-	if x.typecode != y.typecode {
-		// Different dynamic type so always unequal.
+func interfaceEqual(_x, _y interface{}) bool {
```

Comment:
I would strongly suggest adding some tests for this. The file `testdata/interface.go` is probably the most appropriate place at the moment.

Reply:
How do I call unittests locally? Cannot figure this out. Obviously everything is configured for golang on my local machine but in this case the tests need to compile and run using tinygo?!